Python/Python.sublime-syntax

```diff
@@ -37,7 +37,7 @@ variables:
   exponent: (?:[eE][-+]?{{digits}})
   path: '({{identifier}}[ ]*\.[ ]*)*{{identifier}}'
   sql_indicator: \s*(?:SELECT|INSERT|UPDATE|DELETE|CREATE|REPLACE|ALTER|WITH)\b
-  illegal_names: (?:and|as|assert|break|class|continue|def|del|elif|else|except|finally|for|from|global|if|import|in|is|lambda|not|or|pass|raise|return|try|while|with|yield)
+  illegal_names: (?:and|as|assert|break|case|class|continue|def|del|elif|else|except|finally|for|from|global|if|import|in|is|lambda|not|or|pass|raise|return|try|while|with|yield)
```

Comment:
Since `case` is a new keyword, it shouldn't be added here as to not mess up code where `case` is used as an identifier, much like `match` is used in the `re` module. We didn't add the async keywords either for that reason.

Reply:
Yes I noticed that too as an afterthought,